NR-62326: added docs for obfuscation expression (#2091)
newrelic/terraform-provider-newrelic: website/docs/r/obfuscation_expression.html.markdown

---
layout: "newrelic"
page_title: "New Relic: newrelic_obfuscation_expression"
sidebar_current: "docs-newrelic-resource-obfuscation-expressione"
description: |-
Create and manage Obfuscation Expression.
---

# Resource: newrelic\_obfuscation\_expression

> Use this resource to create, and delete New Relic Obfuscation Expression.

Reviewer: ```suggestion
Use this resource to create, update and delete New Relic Obfuscation Expressions.
```

## Example Usage

```hcl
resource "newrelic_obfuscation_expression" "foo"{ 

>   account_id =12345

Reviewer: ```suggestion
  account_id = 12345
```

  name = "OExp"
  description = "The description"
  regex = "(regex.*)"
}
```

## Argument Reference

The following arguments are supported:

* `account_id` - (Optional) The account id associated with the obfuscation expression.
* `description` - (Optional) Description of expression.
* `name` - (Optional) Name of expression.
* `regex` - (Optional) Regex of expression.

## Attributes Reference

In addition to all arguments above, the following attributes are exported:

> * `expression_id` - The id of the obfuscation expression.

Reviewer: ```suggestion
* `id` - The id of the obfuscation expression.
```
